### PR TITLE
Add Alpine image with FIPS inside

### PR DESCRIPTION
**Pull request by lucacome**
### Proposed changes
Adds a new image with FIPS module and necessary configuration. The nginx-plus-module-fips-check was added to the base images. This lightweight module adds a line to the nginx output log printing whether FIPS mode is enabled or not.

### CODE REVIEW

File: internal/configs/version1/nginx-plus.tmpl [hunk 1]
```diff
@@ -21,6 +21,7 @@ load_module modules/ngx_http_app_protect_module.so;
 {{- if .AppProtectDosLoadModule}}
 load_module modules/ngx_http_app_protect_dos_module.so;
 {{- end}}
+load_module modules/ngx_fips_check_module.so;
```
Comment (vepatel) [inline internal/configs/version1/nginx-plus.tmpl:24]: does this adds fips to all the images?

**Comment by vepatel** [comment]
I think we should add a test-run job for alpine-plus-fips image as well
`{\"image\": \"alpine-plus-fips\", \"marker\":\"policies\"}, \`

File: internal/configs/version1/nginx-plus.tmpl (hunk 1 above)
Comment (lucacome) [reply internal/configs/version1/nginx-plus.tmpl:24]: This module only prints the line in the logs saying if FIPS is enabled or not. I've added it to all the plus images (it's only a few KBs)

Comment (shaun-nx) [reply internal/configs/version1/nginx-plus.tmpl:24]: That lines up with what @vepatel showed us in the logs from his testing. Thanks for confirming.

**Review by shaun-nx — APPROVED** [review]

**Pull request merged**